### DISCUSSION TITLE
[Bug report] Simple v3.0.1.0 - miss class="active" in menu type "Post index link"

### DISCUSSION THREAD

**Discussion by Vinc26** [GetPublii/Publii#1749] [Bug Report] [answered]
### Operating system

macOS Sequoia 15.0.1

### Publii version

0.46.1

### Issue type

Application

### Bug description

My homepage is a simple page. In my menu, I've add a link to my blog, using "Post index link" type link.
In the output, there is no "active" class added to the `<li>` tag when I'm on the blog page. I need it to add a custom CSS to bold this li when activated.

### Steps to reproduce

New website
Activate "Use Pretty URLs"
Create a page
Set ti as home page
Create a post
Create a menu
In this menu, add a link type "Post index link"
Preview:
Click on this link
When inspect, there is no class "active" for this li.

**Comment by bjazmoore** [comment]
I do not think this is a bug.  I have never noticed this type of feature when using the Simple theme.  This is probably something that has to be set in the theme.  

By the way you did not mention setting the post prefix to a specific value in your list of steps.  This needs to be done as well:

![1](https://github.com/user-attachments/assets/64272967-1ba8-43e9-8015-12fa3269e8cb)

**Comment by Vinc26** [comment]
When using the Simple theme, you just have to add this custum css
```
.navbar .navbar__menu li.active {
    font-variation-settings: "wght" 700
}
```
And you get the activ menu item in **bold** when it links to a Page. It doesn't work if it links to a post or the blog.

@bjazmoore, yes, I've set the post prefix to "blog"

Any idea to get the blog menu bold when active? Any hack?

**Comment by Vinc26** [comment, marked as answer]
Finally, I've just found a workaround:

- Add a class to the menu item, like "blog"
- In the Custom HTML "After Every post", add: `<style> .navbar .navbar__menu li.blog {font-variation-settings: "wght" 700}</style>`
- In the Them > Layout > Hero Section, add as sample `[BOLDBLOGMENU]`
- Using a plugin like Smart Replacer or Key Replacement, replace `[BOLDBLOGMENU]` by `<style> .navbar .navbar__menu li.blog {font-variation-settings: "wght" 700}</style>`

Then, when you are on the list blog, or in a post, the Blog menu item will be bolded. :D
